airflow/providers/google/cloud/operators/mlengine.py

```diff
@@ -183,6 +185,7 @@ def __init__(self,  # pylint: disable=too-many-arguments
                  project_id: Optional[str] = None,
                  gcp_conn_id: str = 'google_cloud_default',
                  delegate_to: Optional[str] = None,
+                 labels: Optional[Dict[str, str]] = None,
```

Comment:
I'd like to come back through here and update the other `dict` types with more detail like this.